stories/object-attribute/object-attribute.stories.js

```diff
@@ -0,0 +1,26 @@
+export default {
+    title: 'Dev/Object Attribute',
+    components: ['object-attribute', 'button']
+};
+
+export const Text = () => `<div class="fd-object-attribute" id="standardtext">Standard Object Attribute Text</div>
+`;
+
+Text.parameters = { docs: { disable: true } };
+
+export const TextWithLink = () => `<a id="textwithlink" href="#" class="fd-object-attribute--link" role="link" tabindex="0">Text With Link</a>
```

Comment:
`role="link"` is redundant on `<a>`

```suggestion
export const TextWithLink = () => `<a id="textwithlink" href="#" class="fd-object-attribute--link" tabindex="0">Text With Link</a>
```